src/handlers/meta_transaction_handlers.ts

```diff
@@ -171,7 +183,23 @@ export class MetaTransactionHandlers {
                 estimatedGasTokenRefund: ZERO,
                 allowanceTarget: metaTransactionPrice.allowanceTarget,
             };
-            res.status(HttpStatus.OK).send(metaTransactionPriceResponse);
+
+            let priceResponse = metaTransactionPriceResponse;
+            if (params.includePriceComparisons) {
+                const prices = priceComparisonUtils.getPriceComparisonFromQuote(CHAIN_ID, swapVersion, params, {
+                    ...metaTransactionPrice,
+                    buyTokenAddress,
+                    sellTokenAddress,
+                });
+
+                if (prices) {
+                    priceResponse = {
+                        ...metaTransactionPriceResponse,
+                        prices,
```

Comment:
Not sure if the attribute should be `prices` or `comparisons` Any thoughts on this? 🤔

Reply:
would it make sense to call this `priceComparisons` since the query param is `includePriceComparisons `?